### PR TITLE
Gp2 3400 Atlas generic content page

### PR DESCRIPTION
**Pull request by froddd**
Atlas generic content page.

<details>
  <summary>Mobile screenshot</summary>

international trade great_8012_international_content_investment_how-we-can-help_(iPhone 6_7_8)

</details>

<details>
  <summary>Desktop screenshot</summary>

international trade great_8012_international_content_investment_how-we-can-help_

</details>

Couple of other snags also fixed:
- Splitting strapline and intro on Regions page
- Moving footer logo inwards on larger screens

To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [ ] (if there are vulnerable requirements) Upgraded any vulnerable dependencies.
 - [ ] (if updating requirements) Requirements have been compiled.
 - [x] (if adding CSS) CSS have been compiled.
 - [x] (if adding HTML) Change is accessible to the standards we subscribe to.
 - [ ] (if changing content) Locale files have been updated and translated strings have been added if available.
 - [ ] (if adding env vars) Added any new environment variable to vault.
 - [ ] (if adding feature flags) Cleaned up old flags
 - [x] (if changing the UI) Add a printscreen to the PR

### CODE REVIEW

File: investment_atlas/templates/investment_atlas/sector.html [hunk 1]
```diff
@@ -44,70 +33,73 @@ <h1 class="atlas-h--xl atlas-m-b-0">{{ page.title }}</h1>
...
+                    </p>
+                </div>
+            </section>
+        {% endif %}
+
+
+        {% for section in page.downpage_content %}
+            <section>
+                <div class="atlas-container atlas-p-t-l atlas-p-b-s">
+                    {% for block in section.value.content %}
```
Comment (bobmeredith) [inline investment_atlas/templates/investment_atlas/sector.html:55]: Maybe a bit picky but I'd suggest using something other than 'block' as the variable name as it's easy to confuse with django template blocks {% block ... %}.  Could be just me, but I did a double-take.

File: core/sass/atlas/overrides/_forms.scss [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    &:before {
+      top: 3px;
+      width: 28px;
+      height: 28px;
+      color: $magna-blue-deep-20;
+      border-color: inherit;
+    }
+
+    &:after {
+      color: $magna-blue-deep-60;
```
[the remark this answers is not in the document]
Comment (froddd) [reply core/sass/atlas/overrides/_forms.scss:21]: Yeah, it's a good point -- given the tokens are having to be replicated here anyway, I'll rename this.

File: core/templates/atlas/components/footer.html [hunk 3]
```diff
@@ -5,28 +5,33 @@
...
+                           href="{{ header_footer_urls.terms_and_conditions }}">{% trans 'Terms and conditions' %}</a>
                     </li>
                     <li>
                         <a id="footer-contact" href="{{ services_urls.contact_us_international }}">
                             {% trans 'Contact us' %}</a>
                     </li>
-{#                    <li>#}
-{#                        <a id="footer-accessibility" href="{{ header_footer_urls.accessibility }}">Accessibility</a>#}
-{#                    </li>#}
+                    {#                    <li>#}
```
[the remark this answers is not in the document]
Comment (froddd) [reply core/templates/atlas/components/footer.html:28]: No idea, but you're right -- let's be ruthless and not comment things out that aren't needed.

**Comment by stevejalim** [comment]
@froddd  There's also a CTA option available in the main content block - has that been styled for use in the Find A Specialist panel?

File: investment_atlas/templates/investment_atlas/base_cms.html [hunk 4]
```diff
@@ -18,4 +18,27 @@
 {% endblock %}
 {% endcomment %}
 
+{% block body_js %}
+    <script>
+        function atlasToggleReveal(buttonEl, state) {
```
[the remark this answers is not in the document]
Comment (froddd) [reply investment_atlas/templates/investment_atlas/base_cms.html:23]: Good point, will implement this instead.

File: investment_atlas/templates/investment_atlas/opportunity_listing_page.html [hunk 5]
```diff
@@ -1,119 +1,144 @@
...
+        <div class="atlas-container atlas-container--wide atlas-p-v-m">
+            <div class="atlas-grid">
+                <section class="atlas-grid__column atlas-grid__column--4-12-l atlas-grid__column--3-12-xl atlas-m-b-s">
+                    <div class="atlas-bg--white atlas-p-s">
+                        <h1 class="atlas-h--s atlas-m-b-s">Investment opportunities</h1>
 
-{% block css_layout_class %}
-    {% comment %} TO COME DURING FE TICKET {% endcomment %}
-{% endblock %}
+                        {#                        {% if filters %}#}
```
[the remark this answers is not in the document]
Comment (froddd) [reply investment_atlas/templates/investment_atlas/opportunity_listing_page.html:16]: This shows currently selected filters, which is currently unused. Will remove.

File: investment_atlas/templates/investment_atlas/sector.html (hunk 1 above)
Comment (froddd) [reply investment_atlas/templates/investment_atlas/sector.html:55]: Doing a double-take is a good user need!

**Comment by froddd** [comment]
@stevejalim not very usable as it's a different sub-section -- it really needs to live with the 'text' block, as that is where it belongs. Was added to the snags list yesterday!

**Pull request merged**